OpenTabletDriver.Configurations/Configurations/Huion/L310.json

```
@@ -0,0 +1,32 @@
+{
+  "Name": "Huion L310",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 220,
+      "Height": 165.7,

```

This seems incorrect, especially if you consider the officially advertised specs of the tablet. The following change seems necessary:

 ```suggestion
       "Width": 160,
       "Height": 100,
```
This both causes the tablet to match the traditional 200 LPMM ratio, and also seems to match the advertised specifications of the tablet (6.3x3.9 inches, roughly 160x100 cm)